
<launch>
  <arg name="save_path" default="$(find pcd_read)/results/results_1.csv" />
   <!-- Set the file_path to the path where you store the ...map.pcd -->
  <param name="map_path" type="string" value="$(find pcd_read)/map/itri_map_downsample.pcd" />
  <node pkg="pcd_read" type="icp_localization_1" name="ICP_localization_1" output="screen">
    <rosparam param="result_save_path" subst_value="True">$(arg save_path)</rosparam>
  </node>
 <!-- Set the rviz config path to yours  -->
  <node pkg= "rviz" type="rviz" name="rviz" args="-d $(find pcd_read)/rvizConfig.rviz">
    <param name="use_sim_time" value = "true" />
 </node>
    
</launch>